meson.options

```diff
@@ -1,45 +1,70 @@
+# BLAS / LAPACK Selection
 option('blas', type: 'string', value: 'auto',
-        description: 'Option for BLAS library selection. By default, try to find any in the order given by `blas-order`')
+       description: 'BLAS library to use (default: autodetect based on blas-order)')
 option('lapack', type: 'string', value: 'auto',
-        description: 'Option for LAPACK library selection. By default, try to find any in the order given by `lapack-order`')
-option('allow-noblas', type: 'boolean', value: true,
-        description: 'If set to true, allow building with (slow!) internal fallback routines')
+       description: 'LAPACK library to use (default: autodetect based on lapack-order)')
+
 option('blas-order', type: 'array', value: ['auto'],
-       description: 'Order of BLAS libraries to search for. E.g.: mkl,openblas,blis,blas')
+       description: 'Preferred search order for BLAS libraries (e.g., mkl, openblas, blis, blas)')
 option('lapack-order', type: 'array', value: ['auto'],
-       description: 'Order of LAPACK libraries to search for. E.g.: mkl,openblas,lapack')
+       description: 'Preferred search order for LAPACK libraries (e.g., mkl, openblas, lapack)')
+
+option('allow-noblas', type: 'boolean', value: true,
+       description: 'Allow fallback slow internal BLAS implementation if no library is found')
+
 option('use-ilp64', type: 'boolean', value: false,
-       description: 'Use ILP64 (64-bit integer) BLAS and LAPACK interfaces')
+       description: 'Use ILP64 (64-bit integer) BLAS/LAPACK interfaces')
+
 option('blas-symbol-suffix', type: 'string', value: 'auto',
-        description: 'BLAS and LAPACK symbol suffix to use, if any')
+       description: 'Symbol suffix for BLAS/LAPACK symbols (if any)')
+
+
+# MKL / Library Behavior
```

Comment:
This is only a single option so doesn't seem to deserve a separate header; it's part of the BLAS options.